app/schema.graphql

```diff
@@ -23,6 +24,11 @@ enum DataQualityMetric {
   p99
 }
 
+input DataQualityMetricInput {
+  metric: DataQualityMetric!
+  columnName: String
```

Comment:
this works, though maybe an ID system would be better?

Reply:
Yeah I get that this is simpler - but just making it clear that it's a bit of a n inversion of control where you can't enforce the validity of the columnNmae

Reply:
yes, and that is intentional because it aligns with how we treat (missing) columns in general (i.e. just return nan), and unless there's a need for this validation to exist (and it's more like a enhancement than otherwise), we can forgo adding code for it for now

Reply:
I get it, just wanted to highlight that this typically would be something we avoid since it could be a source of user input that we don't trust and not endorsable via rules.